docs/install.md

```diff
@@ -108,7 +108,11 @@ You can also build Nix from source by following the instructions at https://nixo
 	nixops create network/network.nix network/network-vbox.nix -d bitcoin-node
 	```
 
-4. Adjust configuration by opening `configuration.nix` and removing FIXMEs. Enable/disable the modules you want in `configuration.nix`.
+4. Create `configuration.nix` using `sample-configuration.nix`.
```

Comment:
Having a sample-configuration.nix would make it more difficult for us to push updates to configuration.nix like this one. Wouldn't a user have to manually update his configuration.nix from his sample-configuration.nix that isn't .gitignore'd?

Reply:
Yes, that is true.

But, in the case of this PR, an old `configuration.nix` should work without breaking anything or changing any functionality. There are just more options exposed in `configuration.nix`, but all those options have sane defaults and can be ignored, and everything would work as before if they are ignored. I would assume we want to preserve this kind of compatibility with future changes also.

So, without this change, in case of non-breaking changes to `configuration.nix`, users would need to read `configuration.nix` to understand what new options have been added and enable them if they want to. With this change, they would need to read `sample-configuration.nix` and add the options they want to their `configuration.nix`.

(In the case of breaking changes, which I assume we want to avoid, without this change users would probably need to do a merge, and with this change they would get an error on trying to deploy.)

Reply:
I do agree that it might be nice to preserve the option of pushing updates to `configuration.nix` for some things without bothering the user though. My biggest problem with it being tracked by git is it makes switching between branches more tedious, and that I commit it by mistake sometimes. Commiting by mistake is not such a big deal. I could add it to `.git/info/exclude`, but that won't make switching branches easier. I guess `git stash` is the best way to do it, stashing the configuration file separately so I can pop it separately.